README.md

```diff
@@ -1,6 +1,11 @@
-# Landing Page
+# Cambie Collective
 
-## Getting started
+Supporting resilient, inclusive tech and maker communities that empower positive change.
 
-To get going locally, run:
-`npm run develop`
+## Event groups we support
+
+- [NodeSchool YVR](https://nodeschool.io/vancouver/)
+- [Code & Coffee YVR](https://www.meetup.com/codecoffeeyvr/)
+- [PyLadies Vancouver](https://www.meetup.com/PyLadies-Vancouver/)
+- [VanFP](https://www.meetup.com/Vancouver-Functional-Programmers/)
+- [VanJS](https://www.meetup.com/vancouver-javascript-developers/)
```

Comment:
I would just use the suggest changes feature but I don't actually know how to get to it.  can we add 
```
[VanPy](https://www.meetup.com/vanpyz/)
``` 
please?

(I got in touch with everyone before I left on vacation in December, and we actually have the group active again.  Yay!)

Reply:
done.